.github/workflows/snyk-test.yml

```
@@ -0,0 +1,23 @@
+name: Snyk test dependencies
+
+on: 
+  push:
+    branches:
+    - develop
+  pull_request:
+    branches:
+    - develop
+
+jobs:
+  security:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    # Only run on push events or PRs from iotaledger/goshimmer, skip on PRs from forks
    # Secret variables cannot be exposed to PRs from forks
    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
```

Since `secrets.SNYK_TOKEN` is not defined on pull requests from forks, this workflow will likely fail